lib/cassava/client.rb

```diff
@@ -0,0 +1,166 @@
+module Cassava
+  class Client
+    attr_reader :session
```

Comment:
I would be remiss if I didn't not bring this up, do we want ```::CASSAVA_CLIENT.session``` to be part of the public interface? If not might want to bring this down to a private attr_reader. If so we might want to consider sanitizing input and then exposing it directly to a query interface. IE ``::CASSAVA_CLIENT.session_query(""CREATE KEYSPACE test")```